docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md

```diff
@@ -43,6 +43,28 @@ To configure YugabyteDB Anywhere for OIDC, you need to be signed in as a Super A
 - For information on configuring a YugabyteDB Anywhere universe to use OIDC-based authentication using Azure AD as the IdP, refer to [OIDC authentication with Azure AD](../../security/authentication/oidc-authentication-aad/).
 - For information on how to add users, see [Create, modify, and delete users](../anywhere-rbac/#create-modify-and-delete-users). The email ID that you enter in the **Add User** dialog must be registered with the identity provider, and the role must reflect the user's role on YugabyteDB Anywhere.
 
+## Use OIDC groups with YBA roles
+
+If your OIDC provider is configured with user groups, you can map the groups to [YBA roles](../anywhere-rbac/). Users who are members of these groups can then sign in to YBA without needing to be added to YBA first. Users who are members of multiple groups are assigned the most privileged role.
```

Comment:
>YBA roles
We should mention somewhere that we only support built in roles for now and that custom roles will be supported in future releases.